mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/routes/v1/TiltaksgjennomforingRoutes.kt

```diff
@@ -67,6 +76,17 @@ fun Route.tiltaksgjennomforingRoutes() {
             call.respond(tiltaksgjennomforing)
         }
 
+        put() {
+            val request = call.receive<TiltaksgjennomforingRequest>()
+
+            tiltaksgjennomforinger.upsert(request.toDbo())
```

Comment:
Validere at input fra frontend/bruker er korrekt før upserting til db

Reply:
Hadde du noen umiddelbare tanker om hva som burde valideres?
F.eks startDato før sluttdato, sluttdato etter i dag?
antallplasser < x?
Noe annet?

Kanskje ikke noe poeng å sjekke at avtaleId finnes før man prøver å upserte?

Reply:
Hehe, nei hadde ikke det, men syns de du nevner er fine. Spesielt startdato før sluttdato. Det har vært en synder i dataene fra Arena vet vi. 

Hvis du legger inn sjekk på at startdato er før sluttdato og at antall plasser er > 0 så tror jeg vi er good for nå.